### ISSUE TITLE
The labels.csv provided in the repo does not match with the original VRD dataset labels id, can you tell why?

### ISSUE THREAD

**Issue by HimanchalChandra** [pranoyr/visual-relationship-detection#4]

**Comment by pranoyr** [comment]
i just changed motorcycle to motorbike

**Comment by HimanchalChandra** [comment]
The accuracy isn't that great, what was the no. of examples in annotation.csv at the time of training?

**Comment by pranoyr** [comment]
For Predicate Detection, Accuracy is not good. ROI Pooling and some hyperparameters tuning has to be done.

**Comment by HimanchalChandra** [comment]
detections.json is your output of Retinanet Object detection model or ground Truth annotations?

**Comment by pranoyr** [comment]
RetinaNet predictions
